smart-contracts/contracts/mixins/MixinPurchase.sol

```diff
@@ -106,6 +123,19 @@ contract MixinPurchase is
     {
       onKeyPurchaseHook.onKeyPurchase(msg.sender, _recipient, _referrer, _data, inMemoryKeyPrice, pricePaid);
     }
+
+    // refund gas
+    if (_gasRefundPercentage != 0) {
+      uint toRefund = _gasRefundPercentage * pricePaid / 100;
```

Comment:
`/100` will not work. We need to do this in basis points (check methods like the penalties for an example as it is also a %)

Reply:
moved to basis points